woof-distro/x86/debian/bullseye/_00build.conf

```diff
@@ -124,4 +124,5 @@ chroot . /usr/sbin/setup-spot transmission-gtk=true
 mv -f ../adrv/usr/bin/transmission-gtk{,.bin}
 mv -f usr/bin/transmission-gtk ../adrv/usr/bin/
 rm -f usr/bin/transmission-gtk.bin
+echo 'PS1=\"\\\\w\\\\$ \"' > etc/profile.d/ps1.sh
```

Comment:
No need to escape because you are echoing literal with single quotes.
```
 echo "PS1=\"\\w\\\$ \""
PS1="\w\$ "
```

with double quotes: escape`"`escape`\w`escape`\`escape`$ `escape`"` or are you using a different shell?

Reply:
The huge amount of escaping here is needed because we read line by line and `eval` each line in 3builddistro.

Reply:
Ah I see, so I hope you like my 'peaceful' resolution.

Reply:
Definitely! Much cleaner. I thought you were in the 'old prompt camp' too 😊

Reply:
On the fence :person_fencing: 
:rofl: